test/templates/nstemplatetiers/appstudio/spacerole_viewer.yaml

```diff
@@ -0,0 +1,200 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: appstudio-spacerole-viewer
```

Comment:
We don't keep the test templates in host-operator anymore. Let's remove this file.

Reply:
done in a4ff3f73c63567c5d56606e8e8b3f0e8423bae8c